src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -156294,6 +156294,24 @@ public static DateTime Test(DateTime? first, DateTime? second, DateTime? third)
         return DateTime.Now;
     }
 }
+");
+            comp.VerifyDiagnostics();
+        }
+
+        [Fact, WorkItem(61361, "https://github.com/dotnet/roslyn/issues/61361")]
+        public void LiftedRelationalOperation_UserDefined()
+        {
+            var comp = CreateCompilation(@"
+#nullable enable
+class C
```

Comment:
I don't see any lifted operators involved in this test?

Reply:
D'oh. Thanks